src/main/java/be/seeseemelk/mockbukkit/MockBukkit.java

```diff
@@ -25,7 +32,9 @@
 public class MockBukkit
 {
 
+	private static final Logger log = LoggerFactory.getLogger(MockBukkit.class);
 	private static @Nullable ServerMock mock = null;
+	private static final String BUKKIT_PLUGIN_SUPERCLASS = "org.bukkit.plugin.java.JavaPlugin";
```

Comment:
Is there a way for us to get this that's checked at compile time?

Reply:
What do you mean by that? Can you elaborate

Reply:
I was thinking something like `private static final String BUKKIT_PLUGIN_SUPERCLASS = JavaPlugin.class.getName()` instead of storing it as a string. It should™️return the same thing (and I think it can still be final?)